www/data/sql_funcs.json

```diff
@@ -38,6 +38,10 @@
                 "signature": "sum(x: T) -> T",
                 "description": "Sum of `T`'s values"
             },
+            {
+              "signature": "avg(x: T) -> T",
```

Comment:
nit: can you alphabetize this by function name?

Reply:
I'm realizing now `sum` is mis-alphabetized. If you could move that, as well, would be most appreciated.

Reply:
Moved `jsonb_agg` as well. Again, any suggestions on how to express the fact that avg(x: int)-> Decimal, avg(x: real) -> double precision and avg(x: T)-> T otherwise?

Reply:
Could put it in the description:

```
"description": "Average of `T`'s values<br/><br/>Returns `numeric` if `x` is `int`, `double` if `x` is `real`, else returns same type as `x`"

Reply:
Should the signature be changed to something like `avg(x: T) -> U` to signify that the return type does not match the input type?

Reply:
Ah, that's a really good call!